CHANGELOG.md

```diff
@@ -2,6 +2,20 @@
 
 ## [Unreleased]
 
+## [0.34.1] - 2020-04-02
```

Comment:
Not sure why its showing this: these changes are already in `master` https://github.com/DataDog/dd-trace-rb/blob/master/CHANGELOG.md

This branch is based `0.34-stable` (to make a merge and release to 0.34-stable easier without pulling unwanted changes from `master`)... maybe thats part of it?